hubtty/sync.py

```diff
@@ -1106,10 +1097,6 @@ def run(self, sync):
         app = sync.app
         with app.db.getSession() as session:
             change = session.getChange(self.change_key)
-            if change.pending_status_message:
```

Comment:
Same, this should probably be commented out for now, although I'm expecting this part of the code to change quite a bit in the future when we implement different merge strategies.